README.md

```diff
@@ -167,6 +168,42 @@ s.HandleFunc("/{key}/", ProductHandler)
 s.HandleFunc("/{key}/details", ProductDetailsHandler)
 ```
 
+### Listing Routes
+
+Routes on a mux Router can be listed using the Router.Walk method:
```

Comment:
Should describe a use-case - e.g. "Routes on a mux can be listed using the Router.Walk method—useful for generating documentation:"